src/main/haskell/language-kore/test/parser/Data/Kore/ASTVerifier/DefinitionVerifierPatternVerifierTest.hs

```diff
@@ -874,26 +875,28 @@ patternsInAllContexts
     sortVariableName = SortVariableName rawSortVariableName
     symbolAliasSort = sortVariableSort sortVariableName
     symbolSentence =
-        asSentence SentenceSymbol
+        constructUnifiedSentence SentenceSymbolSentence $
+         SentenceSymbol
             { sentenceSymbolSymbol = Symbol
                 { symbolConstructor = testId rawSymbolName
                 , symbolParams = [SortVariable (testId rawSortVariableName)]
                 }
             , sentenceSymbolSorts = [symbolAliasSort]
             , sentenceSymbolResultSort = anotherSort
             , sentenceSymbolAttributes =
-                Attributes [] :: KoreAttributes
+                Attributes [] :: Attributes
```

Comment:
I think you might not need this `:: Attributes` here and elsewhere, as there is only one type of Attributes now.